road map.txt

```diff
@@ -0,0 +1,13 @@
+Create Home pages:
+-
+-
+-
+
+What will be displayed on the landing page?
+
+What problems will be solved?
```

Comment:
Something like web development solutions for people who want to create a web application.